test/modules.coffee

```diff
@@ -457,7 +457,7 @@ test "export class", ->
       baz: ->
         console.log 'hello, world!'"""
   output = toJS input
-  ok /^export (class foo|var foo = \(function)/.test toJS input
+  ok /^export (var foo = class foo|var foo = \(function)/.test toJS input
```

Comment:
We don't name our functions, but we do name our classes. For classes we have always done this.

```coffee
Bar = class Foo
```

```js
Bar = Foo = (function () {
  ...
})()
```

Originally I was not emitting the assignments, however that was breaking `nodes.coffee`, which expects to be able to do `exports.Node = class Node` and still refer to it as `Node`.